src/DisplayDwellings.php

```diff
@@ -28,14 +28,6 @@ public static function displayAllDwellings(Array $dwellingsToDisplay) : string
         $result = '';
 
         foreach($dwellingsToDisplay as $dwelling) {
-            // Check whether the dwelling is still available for purchase/let, as the callout will not
-            // appear in these cases
-            if($dwelling->getStatus()==='Sold' || $dwelling->getStatus()==='Let Agreed') {
-                $statusCapitalized = strtoupper($dwelling->getStatus());
-                $statusBox = "<div class=\"imageStatusText\">$statusCapitalized</div>";
-            } else {
-                $statusBox = '';
-            }
```

Comment:
Totally removing this means the banner appears on all properties, is this intentional?